### ISSUE TITLE
Strawberry GraphQL API - Error handling extra_choices attribute of choice_set object

### ISSUE THREAD

**Issue by svluder** [netbox-community/netbox#17706]
### Deployment Type

Self-hosted

### NetBox Version

v4.1.3

### Python Version

3.12

### Steps to Reproduce

1. Freshly installed / vanilla setup of Netbox 4.1.3
2. Create Custom Field Choice set entry:
- Name: automation
- Choices:
  - enabled:enabled
  - disabled:disabled
3. Create custom field entry:
- Name: device_automation
- Object Types: DCIM > Device
- Type: Selection
- Choice set: automation
- Remaining settings left with default value
4. Run the following GraphQL query in GraphQL client page:
```
{
  custom_field_list
    {
      name,
      choice_set {
        extra_choices
      }
    }
}
```

### Expected Behavior

Output of custom field object device_automation including extra_choices should be shown.

### Observed Behavior

```
{
  "data": {
    "custom_field_list": [
      {
        "name": "device_automation",
        "choice_set": {
          "extra_choices": null
        }
      }
    ]
  },
  "errors": [
    {
      "message": "String cannot represent value: ['enabled', 'enabled']",
      "locations": [
        {
          "line": 6,
          "column": 9
        }
      ],
      "path": [
        "custom_field_list",
        0,
        "choice_set",
        "extra_choices",
        0
      ]
    }
  ]
}
```

**Comment by gellis713** [comment]
@svluder @arthanson I went through the reproduction steps and can confirm this is the same issue as #17562 which is fixed/merged in #17691

**Comment by jeremystretch** [comment]
Thanks @gellis713! Going to close this out as a duplicate.

@svluder we believe this has already been resolved under #17562 and will be fixed in the upcoming v4.1.4 release.
